checks/testdata/Dockerfile-pkg-managers

```diff
@@ -63,4 +63,15 @@ RUN python -m pip install bla3.whl
 
 RUN python -m pip install -r file
 RUN python -m pip install 'some-pkg==1.2.3'
-RUN python -m pip install 'some-pkg>1.2.3'
\ No newline at end of file
+RUN python -m pip install 'some-pkg>1.2.3'
+
+# These first 3 should not result in findings.
+RUN npm install
+RUN npm install -g
+RUN npm i
+
+# These 4 result in findings.
```

Comment:
can you add the same lines to other unit tests, e.g. `checks/testdata/script-pkg-managers`. You can create an additional `checks/testdata/github-workflow-pkg-managers` as well, which seems missing